test/browser/layer-tests/geojson-layer-specs.js

```diff
@@ -19,11 +19,8 @@
 // THE SOFTWARE.
 
 import test from 'tape';
-import GeojsonLayer, {
-  defaultElevation,
-  defaultLineWidth,
-  defaultRadius
-} from 'layers/geojson-layer/geojson-layer';
+import GeojsonLayer, {defaultRadius} from 'layers/geojson-layer/geojson-layer';
+import {DEFAULT_ELEVATION, DEFAULT_LINE_WIDTH} from 'constants/default-settings';
```

Comment:
Please don't move these to default_settings, `defaultElevation` and `defaultLineWidth` is layer specific, even though some layer shared the same vlaue. It still makes sense to keep them under each layer.